templates/benchmark.c.template

```diff
@@ -15,6 +15,10 @@
 #include <unistd.h>
 #include <errno.h>
 
+#ifdef STARCH_COMPAT_REQUIRED
+#  include "compat.h"
```

Comment:
Where does compat.h come from? (I know, it's a dump1090 thing, but in the more general case - is this something that can be supplied as part of starch itself?)

Reply:
I intended for the `compat.h` to be a user-defined header file. The main reason was the fact that you had to use different functions for allocating and freeing aligned memory on the Windows platform. Well to be fair the only discrepancy between major platforms that would break the compilation process is this one and we might as well provide the required wrappers for aligned memory functions within this benchmark.c file or as a separate file. It's up to you whether the user-defined header should stay or I add the required wrappers in this file.

Reply:
I think that providing the windows implementation directly in starch is the way to go here (there's already a bunch of ifdef-ed code there depending on what API is available)

Reply:
Does the new commit resolve this conversation?